test/e2e/README.md

```diff
@@ -0,0 +1,50 @@
+# Pre-requisites
+
+The test doesn't yet manage the lifecycle of the cluster being tested,
+therefore the test hosts' kubeconfig must point to a Kubernetes cluster.
+You can verify that the kubeconfig is setup correctly simply with
+`kubectl get nodes`.
+
+The cluster under test must meet the following requirements:
+* Have 3 nodes
```

Comment:
I believe that's no longer true: https://github.com/openebs/Mayastor/pull/579/files#diff-04c6409df4a44eb62116a9030ccf6bdab282e29f0e57bdc0628f5261f002edbeR8 or at least won't be in the near future when the test is really run, what about raising the requirements right away (as I feel we might forget to update doc later):

```suggestion
* Have 4 nodes
```

Reply:
thanks 👍